package/upgrade/upgrade_manifests.sh

```diff
@@ -245,6 +245,16 @@ upgrade_rancher() {
     exit 1
   fi
 
+  # When upgrading to Harvester v1.3.2, patch Rancher shell-image to v0.1.26 if necessary
+  local shellimage=$(kubectl get settings.management.cattle.io shell-image -ojsonpath='{.value}')
+  if [[ "$shellimage" = "" ]] || [[ "$shellimage" < "rancher/shell:v0.1.26" ]]; then
```

Comment:
Is it possible to meet the following scenario?

```
shellimage=rancher/shell:v0.1.3
if [[ "$shellimage" = "" ]] || [[ "$shellimage" < "rancher/shell:v0.1.26" ]]; then 
  echo need patch
fi
# will not patch
```

I'm not sure if we need to handle all the cases. If this is just a quick fix, assuming the before-upgrade will always use `rancher/shell:v0.1.24`, then we're good.

Reply:
This is a specific patch only on Harvester v1.3.2, suppose it is upgraded from v1.3.0/v1.3.1/v1.3.2, the shell image is  like `v0.1.22`, `v0.1.24`.

Later, shell version will start from v0.2.1.

Reply:
Similar question to the [new cluster case](https://github.com/harvester/harvester-installer/pull/813#issuecomment-2290897219). If we modify the setting to `v0.1.26` here, when the Rancher is upgraded I guess the setting is still v0.1.26 and will need some clean-up.

Reply:
Log a new issue https://github.com/harvester/harvester/issues/6352 to drop this potential patch on v1.3.2 cluster.